packages/dapi/lib/utils/index.js

```diff
@@ -13,6 +13,18 @@ const utils = {
   isDevnet(network) {
     return /^devnet/.test(network);
   },
+  /**
+   * Converts either number or {Buffer} to string in hex
+   * @param bufferOrNumber {number}
+   * @returns {string}
+   */
+  strHexOrNum(bufferOrNumber) {
+    if (typeof bufferOrNumber === 'number') {
+      return bufferOrNumber.toString();
+    }
+
+    return bufferOrNumber.toString('hex');
```

Comment:
You already passing string here. I think you don't need this function you can just pass from to the error message, isn't it?

Reply:
Yeah, I was trying to avoid code duplication, because it was looking messy including this logic in error string

Reply:
Found another way